functional/FunctionalTest.java

```diff
@@ -3880,7 +3909,6 @@ public static void main(String[] args) throws Exception {
       region = args[3];
     }
 
-    isSecureEndpoint = endpoint.toLowerCase(Locale.US).contains("https://");
```

Comment:
If `endpoint` is a passed argument, we should have this.

Reply:
The actual fix is not removing this line and ignore using `endpointTLS` down for this case.